src/main/java/org/traccar/model/ExtendedModel.java

```diff
@@ -154,4 +154,7 @@ public long getLong(String key) {
         }
     }
 
+    public void removeAttribute(String key) {
```

Comment:
If we're adding a new method like this, we should update all the existing places where we could use it currently. We should do a separate PR for that.